API_Products/controllers/publisher_controller.py

```diff
@@ -33,23 +47,60 @@ def read_all_publishers() -> dict:
 
 
 def update_publisher(dict_values) -> dict:
+    """
+    Método para alterar dados das editoras.
+
+    Verifica se os dados enviados são válidos. Retorna dados de erro caso estejam inválidos;
+    Verifica se a editora já está cadastrada. Retorna dados de erro caso já esteja;
+    Verifica se o país está cadastrado. Retorna dados de erro caso não esteja;
+    Altera os dados da editora nos livros caso ela esteja sendo usada;
+    Altera os dados da editora. Retorna mensagem de sucesso.
+    Caso ocorra algum erro ao salvar, retorna dados de erro.
+
+    :param dict_values: {_id, name, country}
+    :return: dict or raise Exception
+    """
+    received = set(dict_values.keys())
+    expected = {"_id", "name", "country"}
+    if expected.symmetric_difference(received) or not dict_values["_id"] or \
+            not dict_values["name"] or not dict_values["country"]:
+        return dict(status=400, error="Valores inseridos inválidos.", message="Verifique os dados informados.")
+
```

Comment:
Sugestão:
Trocar **expected.symmetric_difference(received) or not dict_values["_id"] or not dict_values["name"] or not dict_values["country"]**
Por: **received != expected**
Que da o mesmo resultado.